### PR TITLE
CAT-1940 StopScripts Brick added

### PR DESCRIPTION
**Pull request by robertriedl**
reopens [#1806](https://github.com/Catrobat/Catroid/pull/1806)

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/content/bricks/StopScriptBrick.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+	@Override
+	public int getRequiredResources() {
+		return NO_RESOURCES;
+	}
+
+	@Override
+	public Brick copyBrickForSprite(Sprite sprite) {
+		StopScriptBrick copyBrick = (StopScriptBrick) clone();
+		return copyBrick;
```
Comment (sjaindl) [inline catroid/src/main/java/org/catrobat/catroid/content/bricks/StopScriptBrick.java]: can also be done in 1 line

File: catroid/src/main/res/layout/brick_stop_script.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <Spinner
+            android:id="@+id/brick_stop_script_spinner"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clickable="false"
+            android:focusable="false" />
+
+    </org.catrobat.catroid.ui.BrickLayout >
+
+</LinearLayout>
```
Comment (sjaindl) [inline catroid/src/main/res/layout/brick_stop_script.xml]: Please add a newline here

File: catroid/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -587,6 +587,11 @@
     <string name="brick_when">When tapped</string>
     <string name="brick_wait">Wait</string>
     <string name="brick_when_nfc">When NFC</string>
+    <string name="brick_stop_script">Stop script/s</string>
```
Comment (sjaindl) [inline catroid/src/main/res/values/strings.xml]: Or better: "Stop script(s)" ? :)

Comment (robertriedl) [inline catroid/src/main/res/values/strings.xml]: Yes, your're right, i had this spelling the first time, but the brackets in the string didn't work in the testcases

**Pull request merged**